tests/test_model_loading.py

```diff
@@ -123,7 +194,10 @@ def __eq__(_, name):  # pylint: disable=E,I
 
     def load(self):
         def filehandler(url):
```

Comment:
I think we already have `from __future__ import annotations` at the start of the module, no?
```suggestion
        def filehandler(url: str | pathlib.Path):
```